web_development_101/introduction_to_web_development.md

```diff
@@ -30,14 +30,14 @@ For more details, Udacity has a great blog post on it: [3 Web Dev Careers Decode
 
 Now that you know about the different types of web developers, let's cover what we mentioned earlier about the different types of clients/employers you may work with.
 
-There are large tech companies like Google, Facebook, Amazon, etc. They have very high hiring requirements but they offer excellent pay, benefits, and opportunities.
+There are large tech companies, like Google, Facebook, Amazon, etc. They have very high hiring requirements, but they offer excellent pay, benefits, and opportunities.
 
-There are startups that are a bit like the wild west, for a junior developer it can feel like a trial by fire due to the pace of development. They often offer slightly lower salaries, longer hours, but equity upside and highly unique environments.
+There are startups, which are a bit like the wild west. For a junior developer, it can feel like a trial by fire because of the pace of development. They often offer slightly lower salaries and require longer hours, but also equity upside and highly unique environments.
```

Comment:
Let's change the bit at the end to say `but may also offer equity in the company and highly unique environments`. It will clear up the meaning of this sentence.